### PR TITLE
Fix FAQ about licensing and name of the language.

### LINKED ISSUE #765: Improve a few faqs
This faq:

http://ocaml.org/learn/faq.html#UnderwhatlicensingtermsistheOCamlsoftwareavailable

should mention OCaml's new license terms.

### PR DESCRIPTION
**Pull request by dbuenzli**
closes #765.

### CODE REVIEW

File: site/learn/faq.md [hunk 1]
```diff
@@ -20,11 +20,11 @@ and on PCs under Windows. A brief tour on main
...
 
-The OCaml system is open source software: the compiler is distributed
-under the terms of the Q Public License, and its library is under LGPL;
-please read the [license](/docs/license.html) document for more details. A
-BSD-style license is also available for a fee through the [OCaml
-Consortium](/consortium/).
+The OCaml system is open source software. Since version 4.03 the
+compiler and the standard library are distributed under LGPL 2.1 with
+static linking exception, read the [license](/docs/license.html) for
+details. The sotware is also available under a BSD-style license for a
```
Comment (pw374) [inline site/learn/faq.md:26]: sotware->software

And grammatically speaking, it should be "This piece of software" but it's possible that this rule of grammar is obsolete, I don't know... the English language evolves in a very weird way.

Comment (agarwal) [reply site/learn/faq.md:26]: I fixed the spelling error in 19ce17399a9f5edbe67cd4c73ed94cd42cd83e25. I think "The software" is acceptable now on.

**Comment by agarwal** [comment]
@dbuenzli Thanks for the fixes.

File: site/learn/faq.md (hunk 1 above)
Comment (dbuenzli) [reply site/learn/faq.md:26]: Sorry I missed the notification. I actually just realized that but you beat it to me. Thanks for the fix and the merge.

**Pull request merged**